app/controllers/posts_controller.rb

```diff
@@ -15,12 +15,11 @@ def new
   end
```

Comment:
- [ ] Great job working on this milestone.

To solve the N+1 problem, you were to do so in the `posts#index` actions of the `posts` controller and you could use `@posts = @user.posts.includes(:comments)`